lib/handshake_manager.go

```diff
@@ -1,168 +0,0 @@
-package lib
```

Comment:
Love this deletion. I assume the logic is now embedded in one of the other parts of the code but will keep reading.